cmd/mimir/help-all.txt.tmpl

```diff
@@ -865,6 +867,8 @@ Usage of ./cmd/mimir/mimir:
     	The maximum number of active series per tenant, across the cluster before replication. 0 to disable. (default 150000)
   -ingester.metadata-retain-period duration
     	Period at which metadata we have not seen will remain in memory before being deleted. (default 10m0s)
+  -ingester.out-of-order-allowance value
```

Comment:
Nit: why this says `value` instead of `duration`? :thinking:

Reply:
Because we use `model.Duration`, but `flag` package only knows about `time.Duration` being "duration" :-(

Reply:
We use `model.Duration` to support extended duration formats (years, weeks, days), and use it at many places (most notably in `limits.go` in `pkg/util/validation` package. I would keep `model.Duration` here as well for consistency, although we probably don't need years/weeks durations (but days could be handy).

Reply:
I see, I thought we were using `model.Duration` just because that's what underlying tsdb required.